### ISSUE TITLE
In light mode project name text color contrast with background color is 2.387:1, which is less than 4.5:1:A11y_.NET maui_User can get all the insights of Dashboard_TextContrast

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#30800]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #WCAG1.4.3; #Contrast; #BM_.NETMaui_Win32_JUL2025; #DesktopApp; #Win32; #FTP; #A11ySev2; #.NET MAUI SDKs; #AILimited;

## Environment Details:
Application: Developer Balance (.NET Maui)
Windows 11
OS Build: 26100.2161
Version 24H2

## Repro Steps:

1. Install and open the Developer Balance app
2. TAB till Project tile 
3. Observe the issue 

## Actual Result:
In light mode project name text color contrast with background color is 2.387:1, which is less than 4.5:1

## Expected Result:
Text color contrast with background color should be equal or greater than 4.5:1.

## User Impact:
Low vision user will face difficulty to see the project name. 

## Attachment

<img width="1279" height="716" alt="Image" src="https://github.com/user-attachments/assets/ab746b1d-c29e-4492-b858-59edbb940c39" />

### Repro in MAC platform

<img width="1437" height="815" alt="Image" src="https://github.com/user-attachments/assets/58102d97-5432-4a67-b89f-2b80b235ea4b" />

**Comment by ghost** [comment]
This issue has been verified in Visual Studio 17.14.0 Preview 4.0 with MAUI versions (9.0.0 & 9.0.90). It can be reproduced on Windows platform.

**Comment by jsuarezruiz** [comment]
@sheiksyedm I noticed this has been reopened, could you share the current status? Let me know if there’s any way I can help.

**Comment by kapilvaishna** [comment]
issue is not repro in below version 

<img width="256" height="97" alt="Image" src="https://github.com/user-attachments/assets/df08ccb5-6e25-4899-9d01-9df0d8d3b48f" />


<img width="1280" height="768" alt="Image" src="https://github.com/user-attachments/assets/b855ad28-189b-4904-a7e7-43589b5b6405" />

Not repro in MAC version 9

<img width="1250" height="675" alt="Image" src="https://github.com/user-attachments/assets/59007d95-eecd-465a-8116-c979d62cf78e" />
